docs/development.rst

```diff
@@ -163,6 +163,32 @@ virtual env and run::
 
   pip install -r requirements.d/development.txt
 
+Building on Windows
++++++++++++++++++++
+
+Install scoop: https://scoop.sh/
+
+install MSYS2: ``scoop install msys2``
+
+Use `Mingw64-w64 64bit Shell`.
+In Windows shell activate mingw in the repo root folder: ``mingw``
+
+Then install dependencies
+
+  pacman -S mingw-w64-x86_64-python3 git mingw-w64-x86_64-lz4 mingw-w64-x86_64-python3-pip \
+  mingw-w64-x86_64-cython mingw-w64-x86_64-gcc mingw-w64-x86_64-ntldd-git \
+  mingw-w64-x86_64-pkg-config mingw-w64-x86_64-openssl openssl-devel
+
+(and wait ...)
+
+Use git to get the source and checkout `windows-1.2` branch then::
+
+  pip install -r requirements.d/development.txt
+  ./scripts/win-build.sh
+
+Your compiled file ``borg.exe`` with dependencies is in the dist folder of the repo root.
+Switch to dist folder in your Windows shell and start execute ``borg.exe``.
```

Comment:
Is that borg.exe made by PyInstaller? If so, then please rather say "Your PyInstaller-made borg.exe ...." to not cause wrong expectations (bundling stuff together is different from fully compiling python code to machine code).

Reply:
I'll correct this.